### PR TITLE
gatts_server_service_table: Restart advertising after client disconnect

### PR DESCRIPTION
**Pull request by RobinCutshaw**
Also, cleanup white space, fix log levels and add more debug logging details.

### CODE REVIEW

File: examples/bluetooth/gatt_server_service_table/main/gatts_table_creat_demo.c [hunk 1]
```diff
@@ -113,6 +113,60 @@ static struct gatts_profile_inst heart_rate_profile_tab[HEART_PROFILE_NUM] = {
     
 };
 
+#define ESP_GAP_BLE_NB  ESP_GAP_BLE_SET_PKT_LENGTH_COMPLETE_EVT+1
+static char *esp_gap_ble_cb_event_labels[ESP_GAP_BLE_NB] = {
+    [ESP_GAP_BLE_ADV_DATA_SET_COMPLETE_EVT]          = "ESP_GAP_BLE_ADV_DATA_SET_COMPLETE_EVT",
```
Comment (TianHao-Espressif) [inline examples/bluetooth/gatt_server_service_table/main/gatts_table_creat_demo.c]: These lable tables could be defined as `const`.

**Comment by projectgus** [comment]
I've applied @TianHao-Espressif 's suggestion and pushed these changes into our internal review & merge queue. Thanks @RobinCutshaw for your patience with us.

**Comment by projectgus** [comment]
Hi @RobinCutshaw ,

The Bluetooth team have made a bunch of other changes and cleanup in this example that looks like it includes some of this PR. Is there anything else in this PR that you think should be merged to master?

Angus

**Comment by RobinCutshaw** [comment]
Looks like you picked the one liner fix from my pull request.  I added labels for more detail but that is not required for the fix.  You can close this.

**Comment by projectgus** [comment]
Thanks @RobinCutshaw. Sorry about the mixup and delays on this one.

**Pull request closed without merging**